### ISSUE TITLE
Eltako FCO2TS (EEP A5-09-04)

### ISSUE THREAD

**Issue by tomknocke** [Jey-Cee/ioBroker.enocean#197]
Eltako:
FCO2TS
[**Link to product**:](https://www.eltako.com/de/product/professional-smart-home/das-eltako-sensoren-programm/fco2ts-wg/)

Can't find this devise. Any chance to add it? Basically its Temp and Hum. like others but with CO2 in addition.
If helpful i can extract the raw telegram. Does your adapter support such function? Otherwise is will connect other Monitor temporary.

Thanks.

**Comment by Jey-Cee** [comment]
Yes it is possible to add it. I am actual on a bigger rework, so it takes some time until the next release will come.
